requirements.txt

```diff
@@ -1,4 +1,4 @@
 pynacl>=1.4.0
-discord.py>=1.7.3
-tinytag==1.7.0
+nextcord>=2.0.0a6
```

Comment:
We should install the `voice` package group and drop `pynacl` from the list. `nextcord[voice]` installs pynacl, though it may require more things in the future as well. `pynacl` was only included explicitly, as I had done so in a previous discord bot project - possibly back before `discord.py` had a `voice` package group.

As for the [`speed` group](https://github.com/nextcord/nextcord#installing), I see little downside to it. I tested it, and everything continues to work fine.

The only reason I can think that it'd be optional is that some of the libraries aren't as widely supported across all platforms. In which case, we should probably recommend people install it in our docs (via `pip install nextcord[speed]`) but not require it here.